### PR TITLE
feat(frontend): create multiple todos at once

### PR DESCRIPTION
**Pull request by Haskaris**
Issue ref  #5729

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
~~- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)~~ See #5726 
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/ToDo/ToDoManagement.jsx [hunk 1]
```diff
@@ -7,29 +7,137 @@ import ToDoItem, {
...
-          memberList={props.memberList}
-          onClickCancel={handleClickCancel}
-          onClickSaveNewToDo={handleClickSaveToDo}
-        />
+        <div className='toDoManagement__creation'>
+          <div className='toDoManagement__creation__btnSwitch'>
+            <BtnSwitch
+              activeLabel='Creation multiple'
+              checked={isMultiCreationMode}
+              inactiveLabel='Creation unique'
```
Comment (gbonaspetti) [inline frontend_lib/src/component/ToDo/ToDoManagement.jsx:69]:
```suggestion
              activeLabel={props.t('Creation multiple')}
              checked={isMultiCreationMode}
              inactiveLabel={props.t('Creation unique')}
```

**Review by gbonaspetti — APPROVED** [review]

**Review by wwalden — APPROVED** [review]

**Comment by wwalden** [comment]
tested on branch

File: frontend_lib/src/component/Button/LinkButton.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import { darkenColor } from '../../helper.js'
+
+require('./IconButton.styl')
```
Comment (gbonaspetti) [inline frontend_lib/src/component/Button/LinkButton.jsx:7]:
```suggestion
require('./LinkButton.styl')
```

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  useEffect(() => {
+    return tinymceRemove(wysiwygIdSelector)
+  }, [])
+
+  return (
+    <CardPopup
+      customClass='createToDoFromTextPopup'
+      customColor={props.customColor}
+      onClose={props.onClickClose}
+      label={props.t('Transorm todos')}
```
Comment (gbonaspetti) [inline frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx:23]:
```suggestion
      label={props.t('Transform to tasks')}
```

Comment (gbonaspetti) [reply frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx:23]: Or 'Create task from text' like the button

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/ToDo/ToDoManagement.jsx [hunk 4]
```diff
@@ -1,39 +1,224 @@
...
+    tmpToDoList[index].value = e.target.value
+    setNewToDoList(tmpToDoList)
+  }
+
+  const handleChangePopUpValue = (e) => {
+    setNewToDoListAsText(e.target.value)
+    const lines = e.target.value.split(/\n/g)
+    const tmpToDoList = []
+    lines.forEach((line, index) => {
+      // NOTE - MP - 2022-07-04 - This regex will:
```
[the remark this answers is not in the document]
Comment (Haskaris) [reply frontend_lib/src/component/ToDo/ToDoManagement.jsx]: Done in 9d50a2d

File: frontend_lib/src/component/ToDo/NewToDo.jsx [hunk 5]
```diff
@@ -1,82 +1,58 @@
...
-        <IconButton
-          text={props.t('Validate')}
-          icon='fas fa-check'
-          onClick={() => props.onClickSaveNewToDo(assignedUserId, newComment)}
-          disabled={!newComment}
-          color={props.customColor}
-          intent='primary'
-          mode='light'
+        <span>
+          {props.t('Enter your To Do bellow:')}
```
[the remark this answers is not in the document]
Comment (Haskaris) [reply frontend_lib/src/component/ToDo/NewToDo.jsx]: Done in 9d50a2d

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx (hunk 3 above)
Comment (Haskaris) [reply frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx:23]: Done in 9d50a2d 
I have chosen `Create task from text`

File: frontend_lib/src/component/Button/LinkButton.jsx (hunk 2 above)
Comment (Haskaris) [reply frontend_lib/src/component/Button/LinkButton.jsx]: Done 9d50a2d

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
```
Comment (gbonaspetti) [inline frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx:23]: Up to change this title

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+          onClick={props.onClickTransform}
+          text={props.t('Create task from text')}
+          type='button'
+        />
+      </div>
+    </CardPopup>
+  )
+}
+
+export default translate()(appContentFactory(TracimComponent(CreateToDoFromTextPopup)))
```
Comment (gbonaspetti) [inline frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx:61]: Do you need appContentFactory and TracimComponent?

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx (hunk 7 above)
Comment (Haskaris) [reply frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx]: Indeed, I don't need them
Removed in 2029211

File: frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx (hunk 6 above)
Comment (Haskaris) [reply frontend_lib/src/component/ToDo/CreateToDoFromTextPopUp.jsx]: Done in 2029211

**Review by gbonaspetti — APPROVED** [review]

**Comment by wwalden** [comment]
tested on branch

**Pull request merged**